src/storm/solver/helper/ValueIterationOperator.h

```diff
@@ -211,12 +224,39 @@ class ValueIterationOperator {
         return {(*offsets.first)[offsetIndex], offsets.second};
     }
 
+    template<OptimizationDirection RobustDirection, typename OpT, typename OffT>
+    OpT robustInitializeRowRes(std::vector<OpT> const&, std::vector<OffT> const& offsets, uint64_t offsetIndex) const {
+        return offsets[offsetIndex].upper();
+    }
+
+    template<OptimizationDirection RobustDirection, typename OpT1, typename OpT2, typename OffT>
+    std::pair<OpT1, OpT2> robustInitializeRowRes(std::pair<std::vector<OpT1>, std::vector<OpT2>> const&, std::vector<OffT> const& offsets,
+                                                 uint64_t offsetIndex) const {
+        return {offsets[offsetIndex].upper(), offsets[offsetIndex].upper()};
+    }
+
+    template<OptimizationDirection RobustDirection, typename OpT1, typename OpT2, typename OffT1, typename OffT2>
+    std::pair<OpT1, OpT2> robustInitializeRowRes(std::pair<std::vector<OpT1>, std::vector<OpT2>> const&,
+                                                 std::pair<std::vector<OffT1> const*, OffT2> const& offsets, uint64_t offsetIndex) const {
+        return {(*offsets.first)[offsetIndex], offsets.second};
```

Comment:
Why do we not need `.upper()` here as well?

Reply:
This is dead code and likely not correct. I added error messages in case it gets called.